PythonRegression/tests/features/steps/api_test_steps.py

```diff
@@ -124,23 +124,93 @@ def compare_thread_return(step,apiCall):
     logger.info('Responses match')
 
 
-@step(r'GTTA is called (\d+) times on "([^"]+)"')
-def spam_call_gtta(step,numTests,node):
-    apiCall = 'getTransactionsToApprove' 
-    world.config['apiCall'] = apiCall
+#TODO: Break these functions down into utilities
+@step(r'GTTA is called (\d+) times')
+def spam_call_gtta(step,numTests):
+    start = time()
+    node = next(iter(world.machine['nodes']))
+    apiCall = 'getTransactionsToApprove'
+
     world.config['nodeId'] = node
-    
-    api = api_utils.prepare_api_call(node)
-    logging.info('Calls being made to %s',node)
-    responseVal = []
-    for i in range(int(numTests)):
-        logging.debug("Call %d made", i+1)
+    world.config['apiCall'] = apiCall
+
+    nodes = {}
+    queue = {}
+    threads = {}
+    thread_status = {}
```

Comment:
These should only be in the beginning (even though I don't think you need the `threads` and `thread_status` vars). See later comment.